.flowconfig

```diff
@@ -1,28 +1,18 @@
 [ignore]
```

Comment:
Is this the latest default version of `.flowconfig` provided by the Flow project?